setup.py

```diff
@@ -78,6 +78,7 @@ def run(self):
     "pydot>=1.2.4",
     "speedcopy>=2.0.1; python_version < '3.8' and sys_platform == 'win32'",
     "dataclasses==0.7; python_version < '3.7'",
+    "importlib-metadata==1.4; python_version < '3.8'",
```

Comment:
Should it be `>=1.4`? @isidentical

Reply:
We depend on the stdlib version when the python version is greater than that. So for the sake of consistency among all versions, the 3.8's stdlib importlib_metadata package is the equivalent of `importlib-metadata==1.4`.

Reply:
@isidentical Right, but it is still backward compatible, it is just that 1.4 corresponds to the features introduced in 3.8, but we are still using stdlib's importlib-metadata in 3.9, so is there a reason to keep this requirement so strict?

Reply:
The `importlib-metadata` in 3.8 is going to stay always the same, so if anything is broken in `1.4` or if there is any missing feature we wouldn't be benefitting from it. Obviously we can just change it, but I am not really sure what would it gain to extend the range.

Reply:
@isidentical Just relaxing requirements that have no need to be so strict. Ran into some problems in https://github.com/conda-forge/dvc-feedstock/pull/225/files/5e4708490f97d5395e7273c879a280901c92c11a..5cf119bcc0c486819728c2c2f0a469ce8f53a3ba , so was wondering if this strict requirement is needed.